src/BoilerControllerSystem/Boiler.cs

```
@@ -0,0 +1,125 @@
+                BoilerPhases.PrePurgeState(" running in Prepurge BoilerPhases");
+                Thread.Sleep(1000);
+                BoilerPhases.IgnitionState("Running Ignition BoilerPhases");
+                Thread.Sleep(1000);
+                BoilerPhases.OperationalState(" running in Operational BoilerPhases");
+                status = Status.Operational;
+            }
+            else
+            {
+                Console.WriteLine("Toggle and start so that, the state moves to closed and the status goes to lockout!");
+            }
+
+        }
+        catch (Exception e)
+        {
+            logEvent.LogEventData($"Error : {e.Message}", status.ToString());
+        }
+    }
+
+    public void StopBoiler()
+    {
+        try
+        {
+            Console.WriteLine("Boiler state is stopped!");
+            logEvent.LogEventData("Boiler state is stopped!", status.ToString());
+            status = Status.Lockout;
+            interlockSwitch = InterLockState.Open;
+        }
+        catch (Exception e)
+        {
+            logEvent.LogEventData($"Error : {e.Message}", status.ToString());
+        }
+    }
+
+    public void ToggleInterlockSwitch(InterLockState userChoiceToSwitch)
+    {
+        if (userChoiceToSwitch == InterLockState.Closed)
+        {
+            status = Status.Ready;
+            interlockSwitch = InterLockState.Closed;
+            logEvent.LogEventData("Boiler status changed to ready", status.ToString());
+            Console.WriteLine("Boiler status changed to ready");
+        }
+        else
+        {
+            Console.WriteLine("Interlock state is in open state");
+            logEvent.LogEventData("Interlock Switch Not Closed", status.ToString());
+            status = Status.Lockout;
+            interlockSwitch = InterLockState.Open;
+
+        }
+    }
+
+    public void StimulateBoilerError()
+    {
+        try
+        {
+            if (status == Status.Operational)
```

Use Fail Early Code style here.